entity_parser_key_parser_test.go

```diff
@@ -749,6 +750,7 @@ func TestIndexParse(t *testing.T) {
 				ClusteringKeys: nil,
 			},
 			InputIndexName: "SearchByKey",
+			Columns:        []string{},
```

Comment:
What happens if you don't put `Columns` into this literal? You may be able to leave it off.

Reply:
I start to understand what's difference between `[]string{}` and `[]string(nil)` in golang. Why still no idea why golang is designed in this way :(

Reply:
Yes, the two things are different. However the interesting question is: are they different enough to matter? That is why I suggested trying it out -- if you remove the lines, does it still compile and work as expected?

Reply:
The reason I could remove these lines is I changed the default value of `Columns` from `[]string{}` to `[]string(nil)`